<html>
<head>
<title>TestNG:  TestCauHoi_DeThi</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>TestCauHoi_DeThi</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>14/4/0</td>
</tr><tr>
<td>Started on:</td><td>Mon Dec 06 08:10:38 ICT 2021</td>
</tr>
<tr><td>Total time:</td><td>0 seconds (83 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='testModel.CauHoi_DeThiTest.setDeThi_idTest_TC_15()'><b>setDeThi_idTest_TC_15</b><br>Test class: testModel.CauHoi_DeThiTest</td>
<td><div><pre>org.testng.TestException: 
Method CauHoi_DeThiTest.setDeThi_idTest_TC_15()[pri:0, instance:testModel.CauHoi_DeThiTest@7c9d8e2] should have thrown an exception of type class java.lang.Exception
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 24 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace262445056", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace262445056'><pre>org.testng.TestException: 
Method CauHoi_DeThiTest.setDeThi_idTest_TC_15()[pri:0, instance:testModel.CauHoi_DeThiTest@7c9d8e2] should have thrown an exception of type class java.lang.Exception
	at org.testng.internal.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:82)
	at org.testng.internal.TestInvoker.considerExceptions(TestInvoker.java:749)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:634)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>testModel.CauHoi_DeThiTest@7c9d8e2</td></tr>
<tr>
<td title='testModel.CauHoi_DeThiTest.setCauHoi_idTest_TC_09()'><b>setCauHoi_idTest_TC_09</b><br>Test class: testModel.CauHoi_DeThiTest</td>
<td><div><pre>org.testng.TestException: 
Method CauHoi_DeThiTest.setCauHoi_idTest_TC_09()[pri:0, instance:testModel.CauHoi_DeThiTest@7c9d8e2] should have thrown an exception of type class java.lang.Exception
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 24 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace850551034", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace850551034'><pre>org.testng.TestException: 
Method CauHoi_DeThiTest.setCauHoi_idTest_TC_09()[pri:0, instance:testModel.CauHoi_DeThiTest@7c9d8e2] should have thrown an exception of type class java.lang.Exception
	at org.testng.internal.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:82)
	at org.testng.internal.TestInvoker.considerExceptions(TestInvoker.java:749)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:634)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>testModel.CauHoi_DeThiTest@7c9d8e2</td></tr>
<tr>
<td title='testModel.CauHoi_DeThiTest.setCauTraLoiTest_TC_13()'><b>setCauTraLoiTest_TC_13</b><br>Test class: testModel.CauHoi_DeThiTest</td>
<td><div><pre>org.testng.TestException: 
Method CauHoi_DeThiTest.setCauTraLoiTest_TC_13()[pri:0, instance:testModel.CauHoi_DeThiTest@7c9d8e2] should have thrown an exception of type class java.lang.Exception
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 24 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1740846921", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1740846921'><pre>org.testng.TestException: 
Method CauHoi_DeThiTest.setCauTraLoiTest_TC_13()[pri:0, instance:testModel.CauHoi_DeThiTest@7c9d8e2] should have thrown an exception of type class java.lang.Exception
	at org.testng.internal.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:82)
	at org.testng.internal.TestInvoker.considerExceptions(TestInvoker.java:749)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:634)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>testModel.CauHoi_DeThiTest@7c9d8e2</td></tr>
<tr>
<td title='testModel.CauHoi_DeThiTest.setCauTraLoiTest_TC_11()'><b>setCauTraLoiTest_TC_11</b><br>Test class: testModel.CauHoi_DeThiTest</td>
<td><div><pre>org.testng.TestException: 
Method CauHoi_DeThiTest.setCauTraLoiTest_TC_11()[pri:0, instance:testModel.CauHoi_DeThiTest@7c9d8e2] should have thrown an exception of type class java.lang.Exception
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 24 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1473771722", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1473771722'><pre>org.testng.TestException: 
Method CauHoi_DeThiTest.setCauTraLoiTest_TC_11()[pri:0, instance:testModel.CauHoi_DeThiTest@7c9d8e2] should have thrown an exception of type class java.lang.Exception
	at org.testng.internal.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:82)
	at org.testng.internal.TestInvoker.considerExceptions(TestInvoker.java:749)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:634)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>testModel.CauHoi_DeThiTest@7c9d8e2</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='testModel.CauHoi_DeThiTest.setDeThi_idTest_TC_16()'><b>setDeThi_idTest_TC_16</b><br>Test class: testModel.CauHoi_DeThiTest</td>
<td></td>
<td>0</td>
<td>testModel.CauHoi_DeThiTest@7c9d8e2</td></tr>
<tr>
<td title='testModel.CauHoi_DeThiTest.isTrangThaiTest_TC_08()'><b>isTrangThaiTest_TC_08</b><br>Test class: testModel.CauHoi_DeThiTest</td>
<td></td>
<td>0</td>
<td>testModel.CauHoi_DeThiTest@7c9d8e2</td></tr>
<tr>
<td title='testModel.CauHoi_DeThiTest.getCauTraLoiTest_TC_04()'><b>getCauTraLoiTest_TC_04</b><br>Test class: testModel.CauHoi_DeThiTest</td>
<td></td>
<td>0</td>
<td>testModel.CauHoi_DeThiTest@7c9d8e2</td></tr>
<tr>
<td title='testModel.CauHoi_DeThiTest.getCauTraLoiTest_TC_03()'><b>getCauTraLoiTest_TC_03</b><br>Test class: testModel.CauHoi_DeThiTest</td>
<td></td>
<td>0</td>
<td>testModel.CauHoi_DeThiTest@7c9d8e2</td></tr>
<tr>
<td title='testModel.CauHoi_DeThiTest.isTrangThaiTest_TC_07()'><b>isTrangThaiTest_TC_07</b><br>Test class: testModel.CauHoi_DeThiTest</td>
<td></td>
<td>0</td>
<td>testModel.CauHoi_DeThiTest@7c9d8e2</td></tr>
<tr>
<td title='testModel.CauHoi_DeThiTest.setCauTraLoiTest_TC_14()'><b>setCauTraLoiTest_TC_14</b><br>Test class: testModel.CauHoi_DeThiTest</td>
<td></td>
<td>0</td>
<td>testModel.CauHoi_DeThiTest@7c9d8e2</td></tr>
<tr>
<td title='testModel.CauHoi_DeThiTest.setTrangThaiTest_TC_17()'><b>setTrangThaiTest_TC_17</b><br>Test class: testModel.CauHoi_DeThiTest</td>
<td></td>
<td>0</td>
<td>testModel.CauHoi_DeThiTest@7c9d8e2</td></tr>
<tr>
<td title='testModel.CauHoi_DeThiTest.getDeThi_idTest_TC_06()'><b>getDeThi_idTest_TC_06</b><br>Test class: testModel.CauHoi_DeThiTest</td>
<td></td>
<td>0</td>
<td>testModel.CauHoi_DeThiTest@7c9d8e2</td></tr>
<tr>
<td title='testModel.CauHoi_DeThiTest.setCauHoi_idTest_TC_10()'><b>setCauHoi_idTest_TC_10</b><br>Test class: testModel.CauHoi_DeThiTest</td>
<td></td>
<td>0</td>
<td>testModel.CauHoi_DeThiTest@7c9d8e2</td></tr>
<tr>
<td title='testModel.CauHoi_DeThiTest.getDeThi_idTest_TC_05()'><b>getDeThi_idTest_TC_05</b><br>Test class: testModel.CauHoi_DeThiTest</td>
<td></td>
<td>0</td>
<td>testModel.CauHoi_DeThiTest@7c9d8e2</td></tr>
<tr>
<td title='testModel.CauHoi_DeThiTest.setTrangThaiTest_TC_18()'><b>setTrangThaiTest_TC_18</b><br>Test class: testModel.CauHoi_DeThiTest</td>
<td></td>
<td>0</td>
<td>testModel.CauHoi_DeThiTest@7c9d8e2</td></tr>
<tr>
<td title='testModel.CauHoi_DeThiTest.setCauTraLoiTest_TC_12()'><b>setCauTraLoiTest_TC_12</b><br>Test class: testModel.CauHoi_DeThiTest</td>
<td></td>
<td>0</td>
<td>testModel.CauHoi_DeThiTest@7c9d8e2</td></tr>
<tr>
<td title='testModel.CauHoi_DeThiTest.getCauHoi_idTest_TC_01()'><b>getCauHoi_idTest_TC_01</b><br>Test class: testModel.CauHoi_DeThiTest</td>
<td></td>
<td>0</td>
<td>testModel.CauHoi_DeThiTest@7c9d8e2</td></tr>
<tr>
<td title='testModel.CauHoi_DeThiTest.getCauHoi_idTest_TC_02()'><b>getCauHoi_idTest_TC_02</b><br>Test class: testModel.CauHoi_DeThiTest</td>
<td></td>
<td>0</td>
<td>testModel.CauHoi_DeThiTest@7c9d8e2</td></tr>
</table><p>
</body>
</html>